packages/react-native-bridge/.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+	extends: '../../.eslintrc.js',
+	settings: {
```

Comment:
It should be moved to the overrides section for RN files:

https://github.com/WordPress/gutenberg/blob/eaa1be08e278a76774c040414c1691058caaa653/.eslintrc.js#L167-L179

This way we can control it from one place and avoid local ESLint config files.